lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
```

Comment:
@andy-armstrong I'm trying to write Javascript tests instead of coffee script. I need to include the feature JS file inside my new js/spec/instructor_dashboard_spec.js  which is inside coffee/src/instructor_dashboard but requirejs can not find it. This is the url path it looks for
`http://0.0.0.0:39906/suite/lms/include/coffee/src/instructor_dashboard/student_admin.js`
This file is generated by coffee script interpreter. Is there any existing test which uses a js file generated by coffee script interpreter?